lib/utils/getJsonRpcProvider/index.ts

```diff
@@ -1,5 +1,12 @@
+import { INFURA_ID } from "@klimadao/site/lib/constants";
 import { ethers } from "ethers";
+import { urls } from "../../constants";
 
-export const getJsonRpcProvider = () => {
+export const getJsonRpcProvider = (provider: string) => {
```

Comment:
looks like you pass in the provider string but don't actually use it. We can simplify this:
use the params syntax:
https://github.com/KlimaDAO/klimadao/wiki#prefer-params-objects-over-multiple-arguments

And call it `infuraId`

Then the code looks something like:
```ts
if (params.infuraId) {
  return new ethers.providers.JsonRpcProvider(
      `${urls.infuraRPC}${params.infuraId}`
    );
}
return new ethers.providers.JsonRpcProvider(urls.polygonMainnetRpc);
```